pkg/cmd/kafka/acl/delete/delete.go

```diff
@@ -226,11 +256,20 @@ func validateAndSetOpts(opts *aclcmdutil.CrudOptions) error {
 		return opts.Localizer.MustLocalizeError("kafka.acl.common.error.noPrincipalsSelected")
 	}
 
-	opts.PatternType = aclcmdutil.PatternTypeLITERAL
+	// Backwards compatibility:
```

Comment:
This should be moved after the switch in 264.
Current behavior keeps pattern type as literal if we pass `--prefix` flag.

```
./rhoas kafka acl delete --user enda --topic "rhoas-" --prefix -v
```
```
/rest/acls?patternType=LITERAL&permission=ANY&principal=User%3Aenda&resourceName=rhoas-&resourceType=TOPIC
```

Reply:
Oh.. I did refactor if into switch and introduced that bug. Really hate that golanglint forces us to use switch statements even when they do not make sense